tests/driver_ina220/main.c

```diff
@@ -0,0 +1,101 @@
+/*
+ * Copyright (C) 2015 Eistec AB
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup tests
+ * @{
+ *
+ * @file
+ * @brief       Test application for the INA220 sensor driver
+ *
+ * @author      Joakim Gebart <joakim.gebart@eistec.se>
+ *
+ * @}
+ */
+
+#ifndef TEST_INA220_I2C
+#error "TEST_INA220_I2C not defined"
+#endif
+#ifndef TEST_INA220_ADDR
+#error "TEST_INA220_ADDR not defined"
+#endif
+
+#include <stdio.h>
+
+#include "vtimer.h"
+#include "ina220.h"
+
+/* Use the following configuration:
+ *
+ *  - Continuous measurements, both shunt and bus voltage
+ *  - +/- 320 mV Vshunt range
+ *  - 32 V maximum bus voltage
+ *  - 12 bit ADC resolution, no hardware averaging
+ */
+#define CONFIG   (INA220_MODE_CONTINUOUS_SHUNT_BUS | INA220_RANGE_320MV | \
+                  INA220_BRNG_32V_FSR | INA220_SADC_12BIT | INA220_BADC_12BIT)
+#define CALIBRATION (4096)
+#define SLEEP    (100 * 1000U)
```

Comment:
Choosing the calibration value is dependent on the shunt resistor value and the maximum expected current, since the current and power registers can overflow.